test/powershell/Language/Parser/Parsing.Tests.ps1

```diff
@@ -302,3 +302,11 @@ Describe 'expressions parsing' -Tags "CI" {
 Describe 'Hash Expression parsing' -Tags "CI" {
     ShouldBeParseError '@{ a=1;b=2;c=3;' MissingEndCurlyBrace 2
 }
+
+Describe 'Unicode escape sequence parsing' -Tag "CI" {
+    ShouldBeParseError '"`u{110000}"' InvalidUnicodeEscapeSequenceValue 4 # error offset is "`u{>>1<<10000}"
+    ShouldBeParseError '"`u{1234567}"' UnicodeEscapeSequenceMissingTerminator 10 # error offset is "`u{123456>>7<<}"
+    ShouldBeParseError '"`u219z"' InvalidUnicodeEscapeSequence 1
+    ShouldBeParseError '"`u{219z"' UnicodeEscapeSequenceMissingTerminator 7 # error offset "`u{219>>z<<"
+    ShouldBeParseError '"`u{}"' InvalidBracketedUnicodeEscapeSequence 1
+}
```

Comment:
I'll let you decide what you'd like to do.

I'm certain we had such tests using one of our internal frameworks and I'm not too worried about regressions, but it'd be good to get some tests like that into this repo.